eland/ml/ml_model.py

```diff
@@ -367,6 +379,7 @@ def import_model(
         )
         serializer = transformer.transform()
         model_type = transformer.model_type
+        default_inference_config: Dict[str, Any] = {model_type: {}}
```

Comment:
Change to `Mapping[str, Mapping[str, Any]]` to stay consistent with inference_config.